services.go

```diff
@@ -960,3 +960,99 @@ func (s *ServicesService) DeleteSlackService(pid interface{}, options ...OptionF
 
 	return s.client.Do(req, nil)
 }
+
+// CustomIssueTrackerService represents Custom Issue Tracker service settings.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/services.html#custom-issue-tracker
+type CustomIssueTrackerService struct {
+	Service
+	Properties *CustomIssueTrackerServiceProperties `json:"properties"`
+}
+
+// CustomIssueTrackerServiceProperties represents Custom Issue Tracker specific properties.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/services.html#custom-issue-tracker
+type CustomIssueTrackerServiceProperties struct {
+	ProjectURL  string `json:"project_url"`
+	IssuesURL   string `json:"issues_url"`
+	NewIssueURL string `json:"new_issue_url"`
+}
+
+// GetCustomIssueTrackerService gets Custom Issue Tracker service settings for a project.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/services.html#get-custom-issue-tracker-service-settings
+func (s *ServicesService) GetCustomIssueTrackerService(pid interface{}, options ...OptionFunc) (*CustomIssueTrackerService, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/services/custom-issue-tracker", pathEscape(project))
+
+	req, err := s.client.NewRequest("GET", u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	svc := new(CustomIssueTrackerService)
+	resp, err := s.client.Do(req, svc)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return svc, resp, err
+}
+
+// SetCustomIssueTrackerServiceOptions represents the available SetCustomIssueTrackerService()
+// options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/services.html#createedit-custom-issue-tracker-service
+type SetCustomIssueTrackerServiceOptions struct {
+	NewIssueURL *string `url:"new_issue_url" json:"new_issue_url" `
+	IssuesURL   *string `url:"issues_url" json:"issues_url" `
+	ProjectURL  *string `url:"project_url" json:"project_url" `
+	Description *string `url:"description" json:"description" `
+	Title       *string `url:"title" json:"title" `
+	PushEvents  *bool   `url:"push_events" json:"push_events" `
```

Comment:
All these fields definately need the `omitempty` option for both the `url` and `json` tags.